Apps/Admin/Tests/Service/CsvExportServiceTests.cs

```diff
@@ -14,7 +14,7 @@
 //  limitations under the License.
 // -------------------------------------------------------------------------
 
-namespace HealthGateway.Admin.Client.Services
+namespace HealthGateway.Admin.Client.Service
```

Comment:
Namespace for this file should be `HealthGateway.Admin.Server.Services` - it looks like it and the MapperUtil were incorrectly in the Admin.Client namespace.

Reply:
I've raised bug 14521 as I noticed we had a warning and resolved but the namespace and the actual folder are not matching at all.  The namespace should be HealthGateway.Admin.Tests.Service from our standard perspective.